index.html

```diff
@@ -0,0 +1,50 @@
+<!doctype html>
+<html>
+
+<head>
+  <title>Calculator Benchmark</title>
+  <meta charset="utf-8">
+  <meta name="description" content="calculator">
+  <meta name="author" content="Keith Oppel">
+  <link href="https://fonts.googleapis.com/css?family=Roboto:100" rel="stylesheet">
+  <link type="text/css" rel="stylesheet" href="main.css">
+</head>
+
+<body>
+  <div class="calculator-container">
+    <div class="calculator-display">
+      <span class="display">0</span>
+    </div>
+    <div class="calculator-row">
+      <div class="calculator-button"><button value="AC">AC</button></div>
+      <div class="calculator-button"><button value="±">+/-</button></div>
+      <div class="calculator-button"><button value="%">%</button></div>
+      <div class="calculator-opbutton"><button value="/" style="color:#ffffff">÷</button></div>
```

Comment:
Awww, you don't like my hackey trick... hahah. 😆 I remember doing this as I couldn't get the class to apply the white color to the text on the operator buttons. Perhaps I will ask for help from other people working on the calculator and coach que it if multiple people are having the same issue 😃